packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -330,7 +330,6 @@ export default {
   max-height: 100%;
   display: grid;
   grid-template-rows: 50px 70px 1fr;
-  background-color: var(--oc-color-background-default);
```

Comment:
Now the expanding sub-panels create a weird overlay situation during the animation...